docs/launch-with/seqera-platform-launch-with.rst

```diff
@@ -0,0 +1,42 @@
+Seqera Platform
+===============
+
+Dockstore integrates with the `Seqera Platform <https://seqera.io/>`__,
+allowing you to launch Nextflow-based workflows from Dockstore in a variety of
+cloud environments.
```

Comment:
"variety of cloud environments" -> Is this because Seqera lets you choose the cloud provider? If yes, perhaps a little clarification, or just "variety of cloud environments".

Reply:
Yeah, you select if you want to use AWS batch, slurm, etc. I'll add an "for example..."

<img width="1032" alt="image" src="https://github.com/dockstore/dockstore-documentation/assets/3795586/86092b12-de54-41cd-8414-aa3a9cafa6a3">